storm-server/src/main/java/org/apache/storm/scheduler/resource/strategies/scheduling/BaseResourceAwareStrategy.java

```diff
@@ -539,30 +541,34 @@ protected String nodeToRack(RasNode node) {
      * @return a list of executors in sorted order
      */
     private List<ExecutorDetails> orderExecutorsDefault(
```

Comment:
It will be convenient if the even-distribution executor re-sorting was a single method that could be applied to executor list.
This way the upcoming refactoring of the scheduling strategies would be easier.